src/index.html

```diff
@@ -11,7 +11,143 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+       action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+       method="post"
+      >
+      <fieldset class="fieldset"> 
+        <legend>Personal information</legend>
+       
+        <label class="form">
+           Surname:
+          <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
```
3 point in checklist
you need to add only 2 spaces